chapter12/dcox/ex12_01

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/perl
```

Comment:
I did try to create a hash with attribute description and operators that we're used in an eval but failed to get it working in time.

I hope someone else had more luck than I did, the code duplication depresses me.